wrappers/python/setup.py

```diff
@@ -0,0 +1,42 @@
+"""Module setup."""
+
+import os
+import runpy
+from setuptools import find_packages, setup
+
+PACKAGE_NAME = "ursa_bbs_signatures"
+version_meta = runpy.run_path("./{}/version.py".format(PACKAGE_NAME))
+VERSION = version_meta["__version__"]
+
+with open(os.path.abspath("./README.md"), "r") as fh:
+    long_description = fh.read()
+
+
+if __name__ == "__main__":
+    setup(
+        name=PACKAGE_NAME,
+        version=VERSION,
+        author="Hyperledger Aries Contributors",
+        author_email="aries@lists.hyperledger.org",
+        long_description=long_description,
+        long_description_content_type="text/markdown",
+        url="https://github.com/mattrglobal/ffi-bbs-signatures",
+        packages=find_packages(),
+        # include_package_data=True,
+        
+        package_data={
+            "models": [],
+            "": [
+                "libbbs.dll",
+                "libbbs.dylib",
+                "bbs.so",
```

Comment:
I think this should be `bbs.dll`, `libbbs.dylib`, `libbbs.so`

Reply:
Good catch, thanks!